src/components/GordonLimitedAvailability/index.js

```diff
@@ -0,0 +1,42 @@
+import { Grid, Card, CardContent, Button } from '@material-ui/core';
+
+const GordonLimitedAvailability = ({
+  pageName,
+  backToLocation,
+  backToLink,
+  availableTo,
+  ...OtherProps
+}) => {
+  return (
+    <Grid container justifyContent="center" spacing="16">
+      <Grid item xs={12} md={8}>
+        <Card>
+          <CardContent
+            style={{
+              margin: 'auto',
+              textAlign: 'center',
+            }}
+          >
+            <br />
+            <h1>{pageName + ' Unavailable'}</h1>
+            <h4>{'The ' + pageName + ' page is available for ' + availableTo + ' only.'}</h4>
+            <br />
+            <br />
+            <Button
+              color="primary"
+              backgroundColor="white"
+              variant="outlined"
+              onClick={() => {
+                window.location.pathname = backToLink;
+              }}
+            >
+              Back To {backToLocation}
+            </Button>
```

Comment:
Setting `window.location.pathname` causes the entire site to reload and circumvents React Router. Use the `Link` component instead. See the [EventsAttended Component](https://github.com/gordon-cs/gordon-360-ui/blob/344eb36083b0b15db1fe5fdecdf7e5c78cab576f/src/views/EventsAttended/index.js) lines 43-50 for an example.

Reply:
commit incoming